applications/matter_bridge/sample.yaml

```
@@ -94,3 +94,47 @@ tests:
       - nrf54h20dk/nrf54h20/cpuapp
     platform_allow: nrf54h20dk/nrf54h20/cpuapp
     tags: sysbuild matter
+  applications.matter_bridge.nrf54h20.br_ble.smart_plug:
+    sysbuild: true
+    build_only: true
+    extra_args: SB_CONFIG_WIFI_NRF700X=y
+      CONFIG_CHIP_WIFI=y
+      matter_bridge_SHIELD=nrf700x_nrf54h20dk
+      CONFIG_BRIDGED_DEVICE_BT=y
+      matter_bridge_SNIPPET=onoff_plug
+    integration_platforms:
+      - nrf54h20dk/nrf54h20/cpuapp
+    platform_allow: nrf54h20dk/nrf54h20/cpuapp
+    tags: sysbuild matter
+  applications.matter_bridge.nrf54h20.smart_plug:
+    sysbuild: true
+    build_only: true
+    extra_args: SB_CONFIG_WIFI_NRF700X=y
+      CONFIG_CHIP_WIFI=y
+      matter_bridge_SHIELD=nrf700x_nrf54h20dk
+      CONFIG_BRIDGED_DEVICE_SIMULATED=y
+      matter_bridge_SNIPPET=onoff_plug
+    integration_platforms:
+      - nrf54h20dk/nrf54h20/cpuapp
+    platform_allow: nrf54h20dk/nrf54h20/cpuapp
+    tags: sysbuild matter
+  applications.matter_bridge.nrf70ek.br_ble.smart_plug:
```

Using nrf70ek is not correct here and in the variant below. This uses nrf7002dk - ek is an external shield for the nrf5340dk,
```suggestion
  applications.matter_bridge.br_ble.smart_plug:
```